src/enums/compasses.js

```diff
@@ -0,0 +1,35 @@
+import compassN from '../assets/images/compassPhotos/compassN.png';
+import compassNNE from '../assets/images/compassPhotos/compassNNE.png';
+import compassNE from '../assets/images/compassPhotos/compassNE.png';
+import compassENE from '../assets/images/compassPhotos/compassENE.png';
+import compassE from '../assets/images/compassPhotos/compassE.png';
+import compassESE from '../assets/images/compassPhotos/compassESE.png';
+import compassSE from '../assets/images/compassPhotos/compassSE.png';
+import compassSSE from '../assets/images/compassPhotos/compassSSE.png';
+import compassS from '../assets/images/compassPhotos/compassS.png';
+import compassSSW from '../assets/images/compassPhotos/compassSSW.png';
+import compassSW from '../assets/images/compassPhotos/compassSW.png';
+import compassWSW from '../assets/images/compassPhotos/compassWSW.png';
+import compassW from '../assets/images/compassPhotos/compassW.png';
+import compassWNW from '../assets/images/compassPhotos/compassWNW.png';
+import compassNW from '../assets/images/compassPhotos/compassNW.png';
+import compassNNW from '../assets/images/compassPhotos/compassNNW.png';
+
+export {
+  compassN,
+  compassNNE,
+  compassNE,
+  compassENE,
+  compassE,
+  compassESE,
+  compassSE,
+  compassSSE,
+  compassS,
+  compassSSW,
+  compassSW,
+  compassWSW,
+  compassW,
+  compassWNW,
+  compassNW,
+  compassNNW,
+};

```

Comment:
This isn't really an enum as written, it's just a middle-man file reducing the length of our import statements. We want something like:
```
const compassImages = {
    N: compassN,
    NNE: compassNNE,
    NE: compassNE,
    ...
    etc.
}

export default compassImages;
```
Then we can refer to the images as (for example) `compassImages.NE`, or (to fit what you've already got in the WeatherWindCell) `compassImages[directionWind]`.